ts/features/pn/utils/rptId.ts

```diff
@@ -0,0 +1,29 @@
+import * as E from "fp-ts/lib/Either";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { NotificationPaymentInfo } from "../store/types/types";
+import { PaymentNoticeNumber } from "../../../../definitions/backend/PaymentNoticeNumber";
+import { OrganizationFiscalCode } from "../../../../definitions/backend/OrganizationFiscalCode";
+import { getRptIdFromNoticeNumber } from "../../../utils/payment";
+
+const getRptIdFromPayment = (payment: NotificationPaymentInfo | undefined) =>
```

Comment:
What if we replace 
`(payment: NotificationPaymentInfo | undefined)` 
with
`(payment?: NotificationPaymentInfo)`,
do you think it may result in a better syntax?

Reply:
Changed https://github.com/pagopa/io-app/pull/4624/commits/1b220c8302e23e019a7806d26f990939c3ce18ce